syntax = "proto2";
import "game_event.proto";

message Event_FlipCard {
    extend GameEvent {
        optional Event_FlipCard ext = 2010;
    }
    optional string zone_name = 1;
    optional sint32 card_id = 2;
    optional string card_name = 3;
    optional bool face_down = 4;
}
